vitrina/context_processors.py

```diff
@@ -8,3 +10,17 @@ def current_domain(request):
        "current_domain_full": request.build_absolute_uri("%s://%s" % (protocol, domain)),
        "current_domain": domain,
     }
+
+
+def current_language(request):
+    if request.GET.get('language'):
+        language = request.GET.get('language')
+    elif request.session.get('LANGUAGE_CODE'):
+        language = request.session.get('LANGUAGE_CODE')
+    else:
+        language = settings.LANGUAGE_CODE
+    request.session['LANGUAGE_CODE'] = language
+    request.session.modified = True
+    return {
+        "LANGUAGE_CODE": language
+    }
```

Comment:
Please, use Django built-in functionality:

https://docs.djangoproject.com/en/3.2/topics/i18n/translation/#the-set-language-redirect-view

Reply:
Is there a reason adding a custom solution for switching language instead of using Django built-in functionality? As I mentioned in the comment above?